readme.md

```diff
@@ -49,22 +57,38 @@ stores an integer representing what order that song appears on the playlist.
 (e.g. a song may be first on one playlist, but 10th on another... that info
 would be stored on the join table.)
 
-## Join Models & Tables
+### Join Models
 
 In rails, we should always create a model to represent our join table. The name
-can technically be anythign we want, but really the model name should be as
+can technically be anything we want, but really the model name should be as
 descriptive as possible, and indicate that it represents an *association*.
 
-Some examples:
+### Think/Pair/Share - 3/2 (5 min)
+
+Turn to your Neighbor and discuss the following for each example:
+
+1. Is this relationship a Many-to-Many or One-to-Many?
+2. If a Many-to-Many, what might we call the Join Model/Join Table?
+
+Examples:
+
+* Users & Events
+* Users & Courses
+* Events & Locations  
+* Photos and Groups
+
+>Many-to-Many Examples:
+* To join posts to categories, we might have a `CategoryEntry` or `Categorization` model
+* To join songs to playlists, we might have a `PlaylistEntry` model
 * To join users and events, we might create an `Attendance` model
 * To join users and courses, we might create an `Registration` model
 * To join photos to groups, we might have a `GroupMembership` model
```

Comment:
not sure I get this example. To join `users` to `groups` we might a have  `Membership` model